README.md

```diff
@@ -1 +1,42 @@
-# kubeprober
+English | [简体中文](./README_CN.md)
+
+# KubeProber
+
+## What is KubeProber?
+KubeProber is a diagnostic tool designed for large-scale Kubernetes clusters. It is used to perform diagnostic items in the kubernetes cluster to prove that the functions of the cluster are normal, such as checking whether the entire life cycle of the Pod is normal, throuth by simulating users to deploy a Deployment, Then expand its capacity, and finally delete it; by performing put/delete/get operations on Etcd, to verify whether etcd is operating normally; By resolving the domain name of the public network or intranet, to verify whether Coredns server is normally. At the same time, users can write their own diagnostic items to extend KubeProber, such as simulating users to access your business system deployed on kubernetes to prove the availability of the business; check whether the certificate of domain has expired; verify whether MySQL is running normally through the operation of mysql-client, and so on. KubeProber has the following characteristics:
```

Comment:
too long，too many examples.
examples can be used as an ability to supplement another document.